tests/tlstest.py

```diff
@@ -476,12 +478,13 @@ def connect():
             connection.handshakeClientCert(settings=settings)
             print("%s %s:" % (connection.getCipherName(), connection.getCipherImplementation()), end=' ')
 
-            startTime = time.clock()
+            startTime = timeit.default_timer()
             connection.write(b"hello"*10000)
             h = connection.read(min=50000, max=50000)
-            stopTime = time.clock()
+            stopTime = timeit.default_timer()
+            sizeofdata = sys.getsizeof(h)*2
```

Comment:
`sys.getsizeof()` returns the size of allocated memory for the object, not the length of the string (actually transferred data).

See:
```
>>> import sys
>>> sys.getsizeof(bytearray(0))
48
>>> sys.getsizeof(bytearray(12))
61
```

`len()` is entirely sufficient here

Reply:
Or differently defined and Interpreted:
sys.getsizeof() **returns the size of  an object in bytes.**  The object can be any type of object. All built-in objects will return correct results, but this does not have to hold true for third-party extensions as it is implementation specific.

Whether len() returns **the length (the number of items) of an object.** 

Therefore it seemed more accurate to measure the total size of the actual bytes and not the number of the elements and hence this choice.

Reply:
the "overhead" isn't transmitted - the socket transmits bytes, not objects